src/Microsoft.DotNet.Darc/src/DarcLib/MaestroApiBarClient.cs

```diff
@@ -254,7 +254,7 @@ public async Task<IEnumerable<MergePolicy>> GetRepositoryMergePoliciesAsync(stri
             {
                 return await _barClient.Repository.GetMergePoliciesAsync(repository: repoUri, branch: branch);
             }
-            catch (RestApiException e) when (e.Response.StatusCode == HttpStatusCode.NotFound)
+            catch (RestApiException e) when (e.Response.Status.Equals(HttpStatusCode.NotFound))
```

Comment:
I don't think this will work. Use the same syntax as the other checks `Status == (int) <code>`

Reply:
I changed this, may be I missed this one place.